api/WindowEventHandlers.json

```diff
@@ -293,106 +293,6 @@
           }
         }
       },
-      "onmessage": {
```

Comment:
Another case where I'm not sure where this data went. What feature does this map to? Or if it doesn't exist, why isn't it added, based on this?

(My hunch here is that this PR is both demixing and consolidating events, but that's rather difficult to understand and verify from the diff alone.)

Reply:
For the `WindowEventHandlers` mixin, they have been merged into `Window`!